org.opent2t.sample.hub.superpopular/com.insteon.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,300 @@
+﻿/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
```

Comment:
Fixed.

---
In reply to: [82904867](https://github.com/openT2T/translators/pull/107#discussion_r82904867) [](ancestors = 82904867)